app/initialize.js

```diff
@@ -70,20 +72,24 @@ function renderTable(data) {
   var tableHeader = repoTable.append('thead').append('tr');
   tableHeader.append('th').html('Block')
     .on('click', function() {
-      renderRows(data, tableBody, 'name');
+      ascending = sortedBy === 'name' ? !ascending : false;
```

Comment:
I think this works and could be its own function.

My only other idea at the moment is to make an ascending object that has each column header as a property: 

``` 
var ascending = {

     'name': true,
     'default_branch': false

}
```